packages/gnarly-core/src/models/Transaction.ts

```diff
@@ -37,7 +46,27 @@ export default class Transaction {
     this.methodName = methodAbi.fullName
     this.signature = methodAbi.signature
     this.methodId = methodAbi.shortId
-    // @TODO(shrugs) - do that
+
+    // get abi item
+    const paramAbiItem = registeredAbi.find((item) => item.signature === this.signature)
```

Comment:
instead of "param" for `paramAbiItem` and `paramData`, can we just use the generic `abiItem` and `data` or maybe `transactionAbiItem` and `transactionData`